pkgs/desktops/xfce/art/xfwm4-themes.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  p_name  = "xfwm4-themes";
+  ver_maj = "4.10";
+  ver_min = "0";
+
+  src = fetchurl {
+    url = "mirror://xfce/src/art/${p_name}/${ver_maj}/${name}.tar.bz2";
+    sha256 = "0xfmdykav4rf6gdxbd6fhmrfrvbdc1yjihz7r7lba0wp1vqda51j";
+  };
+  name = "${p_name}-${ver_maj}.${ver_min}";
+
+  meta = {
```

Comment:
I copied ```meta``` from the next file https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/xfce/art/xfce4-icon-theme.nix
Reviewing and fixing xfce livenses may be the topic of another PR, as there are many .nix files in xfce dir lack license. And even those which have license in meta are not consistent (there are gpl2, gpl2plus and gpl3, although they are part of the same project)